### ISSUE TITLE
Explicitly specify migration timestamp column

### ISSUE THREAD

**Issue by sysashi** [bitcrowd/carbonite#108]
Hi, thanks for the awesome library. I'm not entirely sure if this is something I can fix with just proper config or it requires some code changes.

Problem: In my project we use `created_at` timestamp instead of default `inserted_at`, example ecto config -
```
config :my_app, MyApp.Repo,
  migration_timestamps: [
    inserted_at: :created_at
  ]
```

And since carbonite also uses `timestamp()` macro these changes will also apply resulting in broken migration.

I can create a PR if any code change is needed.

Guess naive change would be something like `timestamp(inserted_at: :inserted_at)` in places where timestamp is called, but that does not look pretty

**Comment by maltoe** [comment]
Hi @sysashi,

that is an excellent point you're raising. We probably should not rely on these config options in our migrations. I suggest we replace the `timestamps` calls in `Migrations.V1` and `V3` with explicit `add(...)` calls. `timestamps` call in the schema should be ok, I guess.

Please submit a PR :)

**Comment by maltoe** [comment]
Nevermind, it's done :)

released as v0.14.0

**Comment by sysashi** [comment]
@maltoe thanks for the fixes! I was off already yesterday :(

This really makes my life easier, before bootstrapping project and running migrations were such a struggle!
